src/screens/generationList/generationList.vue

```diff
@@ -36,86 +38,98 @@
             :key="i"
           >
             <td>
-              <div
-                v-if="generations[height-i+1]"
-                class="height"
-              >
-                <RouterLink :to="`/generation/${generations[height-i+1].keyBlock.height}`">
-                  {{ generations[height-i+1].keyBlock.height }}
-                </RouterLink>
-              </div>
-              <FillDummy v-else />
+              <transition name="fade">
+                <div
+                  v-if="generations[height-i+1]"
+                  class="height"
+                >
+                  <RouterLink :to="`/generation/${generations[height-i+1].keyBlock.height}`">
+                    {{ generations[height-i+1].keyBlock.height }}
+                  </RouterLink>
+                </div>
+                <FillDummy  v-else />
```

Comment:
This loader gets loaded only for few moments before data is shown, and doesn't show for most of the time.